deploy/local/nodes.sh

```
@@ -1,10 +1,15 @@
 # Install the enclave binary
-cargo install --locked --path ./crates/cli --bin enclave -f
+# cargo install --locked --path ./crates/cli --bin enclave -f
 
 concurrently \
   --names "ANVIL,NODES" \
   --prefix-colors "blue,yellow" \
   "anvil" \
-  "cd examples/CRISP && enclave wallet set --name ag --private-key "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" && enclave nodes up -v"
\ No newline at end of file
+  "cd examples/CRISP && \
+  enclave wallet set --name ag --private-key "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" && 
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)